en/docs/install-and-setup/setup/deployment-best-practices/changing-the-hostname.md

```diff
@@ -51,6 +51,29 @@ Follow the steps given below.
     <ip_address>    <hostname>
     ```
 
+!!! Note
+    For internal calls APIM will assume the hostname as localhost. If we are in a need to change this, we need to configure the hostname by following below steps.
```

Comment:
Shall we change this to something like

```
For internal calls, APIM assumes the hostname to be localhost. If you need to change this, follow the steps below to configure the hostname:
```

to avoid using terms like "we"?